exercises/satellite/canonical-data.json

```diff
@@ -74,6 +74,42 @@
       "expected": {
         "error": "traversals must contain unique items"
       }
+    },
+    {
+      "uuid": "d232a4e5-622e-4aaa-965e-244ddcdcdb96",
+      "description": "Complex tree with many integer values",
+      "property": "treeFromTraversals",
+      "input": {
+        "preorder": ["2", "1", "3", "6", "5", "7"],
+        "inorder": ["1", "2", "3", "5", "6", "7"]
+      },
+      "expected": {
+        "v": "4",
```

Comment:
there is no 4 in the preorder nor inorder traversals, so the expected answer cannot contain a 4

Reply:
Done in the new commit.